media/js/firefox/accounts.js

```diff
@@ -20,6 +24,18 @@
         fxaIframeSrc = fxaIframeSrc.replace('context=iframe', 'context=fx_firstrun_v2');
     }
 
+    // check for fxaEmail in sessionStorage or URL params
+    try {
+        fxaEmail = sessionStorage.getItem('fxa-email');
+    } catch(ex) {
+        fxaEmail = params.get('fxa-email');
```

Comment:
Hm, agree that we should do more to ensure the email address doesn't make it into the URL. I can check for `sessionStorage` and only show the email field if it's available.

Will also update to add a bit of validation on the email address when pulling from `sessionStorage`.